relay-server/src/extractors/request_meta.rs

```diff
@@ -479,6 +480,30 @@ fn auth_from_parts(req: &Parts, path_key: Option<String>) -> Result<Auth, BadEve
         auth = Some(header.parse::<Auth>()?);
     }
 
+    // try to get authentication info from basic auth
+    if let Some(basic_auth) = req
```

Comment:
What the reason even to add basic auth? 

Using basic auth is super confusion to me to be honest. 

Especially when you look at the format users should use to make it work. And I talk about the part when you have yo prove the key (`curl -u sentry_key: `) this colon after `sentry_key` is super easy to miss and mess up your curl command. IMO it makes this very error-prone to the customer. 

Much easier and more explicit way would be to use the header which we support or path param, which is also supported.

Reply:
We're adding this as a bit of an experiment for crons to feel out the ergonomics and DX doing it this way.

I don't think we're planning to immediately tell customers to use this. I agree with the points you make though @olksdr, I think it's not unlikely this will get removed.

Reply:
Please, add the note here, which explains why this added and that this is just an experiment and will be removed in the follow-up.

Reply:
@olksdr fwiw I think basic auth should be generally supported, beyond this experiment, entirely independent of this. From the original `x-sentry-auth` header basically nothing remains and we already support auth purely by `?sentry_key` in the URL. At this point I would honestly love to see us supporting just `Bearer` and basic auth with just the public key.

cc @jan-auer